db/migrate/20220713214100_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.text :title
+      t.text :text
+      t.integer :likes_counter
+      t.string :timestamps
+
+      t.timestamps
```

Comment:
- [ ] there should be a comments counter on the posts table as well.

Reply:
@mikethreels Thank you for your valuable feedback.